src/Ilios/CliBundle/Command/FixLearningMaterialMimeTypesCommand.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+namespace Ilios\CliBundle\Command;
+
+use Ilios\CoreBundle\Entity\LearningMaterialInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Helper\ProgressBar;
+
+use Ilios\CoreBundle\Entity\Manager\LearningMaterialManager;
+use Ilios\CoreBundle\Service\IliosFileSystem;
+use Symfony\Component\Debug\Exception\ContextErrorException;
+
+/**
+ * Cleanup incorrectly stored mime types for learning materials.
+ */
+class FixLearningMaterialMimeTypesCommand extends Command
+{
+    /**
+     * @var IliosFileSystem
+     */
+    protected $iliosFileSystem;
+    
+    /**
+     * @var LearningMaterialManager
+     */
+    protected $learningMaterialManager;
+    
+    public function __construct(
+        IliosFileSystem $iliosFileSystem,
+        LearningMaterialManager $learningMaterialManager
+    ) {
+        $this->iliosFileSystem = $iliosFileSystem;
+        $this->learningMaterialManager = $learningMaterialManager;
+        
+        parent::__construct();
+    }
+    
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('ilios:maintenance:fix-mime-types')
+            ->setDescription('Cleanup incorrectly stored mime types for learning materials.');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $totalLearningMaterialsCount = $this->learningMaterialManager->getTotalLearningMaterialCount();
+
+        $helper = $this->getHelper('question');
+        $output->writeln('');
+        $question = new ConfirmationQuestion(
+            '<question>Ready to fix ' . $totalLearningMaterialsCount .
+            ' learning materials. Shall we continue? </question>' . "\n",
+            true
+        );
+        
+        if ($helper->ask($input, $output, $question)) {
+            $progress = new ProgressBar($output, $totalLearningMaterialsCount);
+            $progress->setRedrawFrequency(208);
+            $output->writeln("<info>Starting cleanup of learning materials...</info>");
+            $progress->start();
+
+            $fixed = 0;
+            $skipped = 0;
+            $offset = 0;
+            $limit = 50;
+            $errors = [];
+
+            while ($fixed + $skipped < $totalLearningMaterialsCount) {
+                /** @var LearningMaterialInterface[] $learningMaterials */
+                $learningMaterials = $this->learningMaterialManager->findBy([], ['id' => 'desc'], $limit, $offset);
+                foreach ($learningMaterials as $lm) {
+                    $mimetype = $lm->getMimetype();
+                    if ($lm->getCitation()) {
```

Comment:
the conditional is not strong enough. we have a multitude of citations with blank text.